### ISSUE TITLE
Cannot map mouse scroll to Mod

### ISSUE THREAD

**Issue by Jos512** [UltimateHackingKeyboard/agent#830]
I'm using:

* Agent version: 1.2.11
* Firmware 8.2.5
* Windows 7 SP1 64-bit

My problem:

* I mapped the 'Scroll Up' and 'Scroll Down' actions to the Mod + G and Mod + T keys.

What happened:

* Nothing happened when I press Mod + G or Mod + T. :slightly_smiling_face: 
* Other keys that are mapped with Mod do work right (like Mod + H for page down or Mod + Y for page up).

What I expected:

* That Mod can trigger the same scrolling behaviour as the Mouse modifier key can.

Here's what my keymap looks:

![2018-10-12_13-39-17](https://user-images.githubusercontent.com/13403160/46867297-cab7fc80-ce24-11e8-90d3-ecfa1bfb8526.png)

**Comment by mondalaci** [comment]
I don't think this is an Agent / UHK bug. Can you actually scroll upwards / downwards in the region you're testing in?

Please try it with a physical mouse, then retry with your UHK.

**Comment by Jos512** [comment]
Thanks for the quick reply. You beat me to it, because I just came here to say that the problem is solved.

What happened: I made the Mod + Scroll key change on a different key map, which I activated with the star. I thought that star behind its name meant that the key map was active.

But I misunderstood the text that appears in the mouse hover window. Switching the star is not enough by itself. I also needed to unplug the UltimateHackingKeyboard and then re-connect the USB cable before the keymap switched.

That I could switch the star to toggle the key map gave me the impression that the key map switch was done, but I was mistaken.

Sorry for the premature GitHub issue.
